source/sync/configure/enable-sync.txt

```diff
@@ -179,33 +179,27 @@ Procedure
                  -X GET \
                  -h 'Authorization: Bearer <Valid Access Token>'
 
-            Once you have the configuration, add the ``flexible_sync`` object with the
-            following template configuration:
+            For details on the sync configuration, refer to the :ref:`sync-configuration-reference`.
 
-            .. code-block:: json
+         .. step:: Enable Development Mode (optional)
 
-               {
-                 ...
-                 "flexible_sync": {
-                   "state": "enabled",
-                   "database_name": "<Name of Database>",
-                   "client_max_offline_days": <Number>,
-                   "is_recovery_mode_disabled": <Boolean>,
-                   "indexed_queryable_fields_names": <Array of String Field Names>,
-                   "queryable_fields_names": <Array of String Field Names>,
-                   "collection_queryable_fields_names": <Map[String][]String>
-                 }
-                 ...
-               }
+            You may need to enable :ref:`Development Mode <development-mode>` before deploying your changes. 
+            You can use the following command to enable Development Mode:
 
-            For details, refer to the :ref:`sync-configuration-reference`.
+            .. code-block:: shell
 
-         .. step:: Deploy the Sync Configuration
+               curl https://services.cloud.mongodb.com/api/admin/v3.0/groups/{GROUP_ID}/apps/{APP_ID}/sync/config \
+                 -X PUT \
+                 -h 'Authorization: Bearer <Valid Access Token>'
+                 -h "Content-Type: application/json"
+                 -d '{"development_mode_enabled": true}'
+
+         .. step:: Update and Deploy the Sync Configuration
 
             To deploy your changes and start syncing data, send an
             :admin-api-endpoint:`Admin API request
```

Comment:
I'd suggest moving this link to "updates the cluster configuration", so the link text is descriptive of the actual call.